## Big Data Resources

>Resources that can be converted to Gene Networks, Gene Set Libraries, and Bipartite Graphs 

National Cancer Instituteâ€™s Cancer Target Discovery and Development ([CTD2](https://ocg.cancer.gov/programs/ctd2/using-ctd2-data)) 
under [the Broad Institute](https://www.broadinstitute.org/scientific-community/science/programs/csoft/chemical-biology/project-cancer-dependencies-%E2%80%93-ctd2/canc). CTD2 is a cancer phenotypingp roject that seeks to identify novel cancer **drug targets** and novel **biomarkers** for diagnosis of cancer and prediction of drug response. 
[Project Achilles](http://www.broadinstitute.org/achilles).

Biological General Repository for Interaction Datasets [(BioGRID)](http://thebiogrid.org/) is a database of **protein-protein** interactions.

Cancer Cell Line Encyclopedia [(CCLE)](http://www.broadinstitute.org/software/cprg/?q=node/11) is a database of **gene expression**, genotype (copy number and mutation status of genes), and drug sensitivity data for human cancer cell lines.

Catalogue of Somatic Mutations in Cancer [(COSMIC)](http://cancer.sanger.ac.uk/cancergenome/projects/cosmic/) is a database of information about somatic mutations in cancer/cancer *tissue* obtained from curation of relevant **literature** and from **high-throughput sequencing data** generated by the Cancer Genome Project and other cancer profiling projects such as The Cancer Genome Atlas Project.

Universal Protein Resource [(UniProt)](http://www.uniprot.org/) is a database containing diverse types of information about proteins (**domain annotations**).

[DrugBank](http://www.drugbank.ca/) is a manually curated database of information about **drugs** and their targets (*not only protein*).

Encyclopedia of DNA Elements ([ENCODE](https://www.encodeproject.org/)) is **a genome mapping** project that seeks to annotate the human genome with information about genes and elements that regulate gene transcription, such as transcription factor binding sites. ENCODE at [UCSC](http://genome.ucsc.edu/ENCODE/).

Gene Ontology [GO](http://geneontology.org/) annotates genes with biological terms (*molecular functions, biological
processes, and cellular components*) selected from a structured vocabulary.
Gene Reference Into Function ([GeneRIF](http://www.ncbi.nlm.nih.gov/gene/about-generif)) is another gene annotation project that is a manually curated collection of short statements, each describing a function of a gene.

[Reactome](http://www.reactome.org/) is a database of **biomolecular pathways**, there are other pathway databases like Kyoto Encyclopedia of Genes and Genomes ([KEGG](http://www.genome.jp/kegg/)) and [WikiPathways](http://www.wikipathways.org/index.php/WikiPathways).

[KINOMEscan](http://lincs.hms.harvard.edu/explore/10.1038-nchembio.1337/fallahi-sichani-2013/dose_response_grid.html) is a Library of Integrated Network-based Cellular Signatures ([LINCS](http://lincs.hms.harvard.edu/)) project that is quantifying in high-throughput the inhibitory effect of drugs on **kinases**.

The [L1000 Cmapis](http://www.lincscloud.org/l1000/) a Library of **LINCS** project that is profiling *gene expression* changes following pharmacologic or genetic (knockdown or over-expression) perturbation of *cell lines* in high-throughput.

The Gene Expression Omnibus ([GEO](http://www.ncbi.nlm.nih.gov/geo/)) is an alternative source of many *gene expression* datasets contributed by **researchers** studying a many diverse topics.

Mammalian Phenotype Ontology ([MPO](http://www.informatics.jax.org/mgihome/projects/aboutmgi.shtml)) developed by Mouse Genome Informatics (MGI) at the Jackson Labs is a controlled vocabulary of **mouse** phenotype terms that are annotated with the genotypes of the mice that display the phenotype.

Human Endogenous Complexome([HEC](http://www.epicome.org/)) is a high content proteomics dataset hosted by the Nuclear Receptor Signaling Atlas ([NURSA](http://www.nursa.org/nursa/index.jsf)) consisting of 3000 immuno-precipitations followed by mass spectrometry. It can be mined for functional protein-protein by complexes.

Online Mendelian Inheritance in Man ([OMIM](http://www.omim.org/)) is a database of human phenotypes (e.g. **diseases**) with known genetic basis that summarizes the current state of knowledge about a gene or phenotype.

Proteomics Database

- [ProteomicsDB](https://www.proteomicsdb.org/) hosts the results of a recently published study that profiled protein abundance in high-throughput across a range of *cell lines* and *tissues*.
- [The Human Proteome Map](http://www.humanproteomemap.org/) is a similar source of data on protein abundance measured in many cell lines and tissues.

The [Allen Brain Atlas](http://www.brain-map.org/) has generated many **brain-related** datasets, it can be mined for genes that are differentially whole-genome gene expressed in a *region* of the brain or at a developmental *time point*. 

Genotype Tissue Expression Project ([GTEx](http://www.gtexportal.org/home/)) is a genome mapping project that seeks to discover expression quantitative trait loci (eQTL) in the human genome, and provide a list of **gene-SNP pairs**.

The Phenotype-Genotype Integrator [(PheGenI)](http://www.ncbi.nlm.nih.gov/gap/phegeni) on NCBI.
